### ISSUE TITLE
AngularJS not in bower package?

### ISSUE THREAD

**Issue by philjones88** [kentcdodds/genie#23]
Hi,

I'm not able to find the AngularJS uxGenie code in any of the bower package files?

**Comment by kentcdodds** [comment]
Can you `bower install ux-genie`?

**Comment by philjones88** [comment]
I thought looking at the repo that was discontinued and put into genie itself?

**Comment by kentcdodds** [comment]
You are correct. I think I need to do some updating to this library... I may work on it soon. Thanks for bringing this up!

**Comment by philjones88** [comment]
You open to PRs to do the work for you? :)

**Comment by kentcdodds** [comment]
Sure thing!
